src/components/common/StaffPayment.vue

```diff
@@ -106,7 +117,7 @@ export default class StaffPayment extends Vue {
   /** Emit an event indicating whether or not the form is valid. */
   @Emit('valid')
   private emitValid (): boolean {
-    return (this.formValid || this.isWaiveFees)
+    return Boolean(this.formValid || this.isWaiveFees)
```

Comment:
`!!` could also be used in place of `Boolean`, but i'm starting to prefer this approach more. Clarity > all, i think. No action required here at all, just thinking out loud :)